Formula/bioperl.rb

```diff
@@ -0,0 +1,34 @@
+class Bioperl < Formula
+  desc "Perl tools for bioinformatics, genomics and life science"
+  homepage "https://bioperl.org"
+  url "https://cpan.metacpan.org/authors/id/C/CJ/CJFIELDS/BioPerl-1.7.8.tar.gz"
+  sha256 "c490a3be7715ea6e4305efd9710e5edab82dabc55fd786b6505b550a30d71738"
+  license any_of: ["Artistic-1.0-Perl", "GPL-1.0-or-later"]
+  head "https://github.com/bioperl/bioperl-live.git", branch: "master"
+
+  livecheck do
+    url :stable
+    regex(/href=.*?>BioPerl-(\d+\.\d+\.\d+)\.t/i)
+  end
+
+  depends_on "cpanminus" => :build
+  depends_on "perl"
+
+  def install
+    ENV.prepend_create_path "PERL5LIB", libexec/"lib/perl5"
+    system "cpanm", "--self-contained", "-l", libexec, "DBI" unless OS.mac?
```

Comment:
```suggestion
    on_linux { system "cpanm", "--self-contained", "-l", libexec, "DBI" }
```

Reply:
We're moving away from using `on_os` blocks in `def install`.

Reply:
I thought that was mostly/only for things that can use ternary operators? This is the first PR in core that I've seen use `unless`.